### ISSUE TITLE
pip install failed

### ISSUE THREAD

**Issue by chenjr0719** [hz-inova/run_jnb#1]
Hi, I want to install this module with `pip install run_jnb`, but there is some error:

```
Collecting run_jnb
  Using cached https://files.pythonhosted.org/packages/36/42/8ec4db3d2394f462230634be7786e9255d331d0168e8b15435a8e54dd1b5/run_jnb-0.1.13.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-s_5vw53o/run-jnb/setup.py", line 2, in <module>
        import run_jnb
      File "/tmp/pip-build-s_5vw53o/run-jnb/run_jnb/__init__.py", line 1, in <module>
        from .core import possible_parameter, run_jnb
      File "/tmp/pip-build-s_5vw53o/run-jnb/run_jnb/core.py", line 6, in <module>
        from nbconvert.preprocessors import ExecutePreprocessor
    ModuleNotFoundError: No module named 'nbconvert'

    ----------------------------------------
```

This is caused by the wrong `setup.py`, see [this](https://github.com/hz-inova/run_jnb/blob/master/setup.py#L2)

When pip starts to install this module, it tries to import this module and all the dependencies like `nbconvert`.
If I didn't install `nbconvert` manually, this module can't be installed by pip.

I try to remove line 2 in `setup.py` will cause another error because of [line 16](https://github.com/hz-inova/run_jnb/blob/master/setup.py#L16) in `setup.py` try to use `run_jnb.__version__`, but variable `run_jnb` isn't import if line 2 deleted.

Replace version with string will fix all problem and install all dependencies automatically.

**Comment by aplamada** [comment]
Hi @chenjr0719 ,
Thanks for reporting this issue!

The issue should be solved in [v0.1.14](https://github.com/hz-inova/run_jnb/releases/tag/v0.1.14) .
I  used 1. from [here](https://packaging.python.org/guides/single-sourcing-package-version/). 

Can you please give it a try?

**Comment by chenjr0719** [comment]
It worked, thanks for your help.
